src/main/java/com/mongodb/jdbc/MySQLColumnInfo.java

```diff
@@ -0,0 +1,175 @@
+        nullTypes.add("maxKey");
+        nullTypes.add("minKey");
+        nullTypes.add("object");
+        nullTypes.add("regex");
+        nullTypes.add("symbol");
+        nullTypes.add("timestamp");
+        nullTypes.add("undefined");
+    }
+
+    public MySQLColumnInfo(
+            String database,
+            String table,
+            String tableAlias,
+            String column,
+            String columnAlias,
+            String bsonType)
+            throws SQLException {
+        this.database = database;
+        this.table = table;
+        this.tableAlias = tableAlias;
+        this.column = column;
+        this.columnAlias = columnAlias;
+        this.bsonType = bsonType;
+    }
+
+    private static int convertToJDBCType(BsonType bsonTypeEnum) throws SQLException {
+        switch (bsonTypeEnum) {
+            case ARRAY:
+                return Types.ARRAY;
+            case BINARY:
+                return Types.BLOB;
+            case BOOLEAN:
+                return Types.BIT;
+            case DATE_TIME:
+                return Types.TIMESTAMP;
+            case DB_POINTER:
+                return Types.NULL;
+            case DECIMAL128:
+                return Types.DECIMAL;
+            case DOCUMENT:
+                return Types.NULL;
+            case DOUBLE:
+                return Types.DOUBLE;
+            case INT32:
+                return Types.INTEGER;
+            case INT64:
+                return Types.INTEGER;
+            case JAVASCRIPT:
+                return Types.NULL;
+            case JAVASCRIPT_WITH_SCOPE:
+                return Types.NULL;
+            case MAX_KEY:
+                return Types.NULL;
+            case MIN_KEY:
+                return Types.NULL;
+            case NULL:
+                return Types.NULL;
+            case OBJECT_ID:
+                return Types.LONGVARCHAR;
+            case REGULAR_EXPRESSION:
+                return Types.NULL;
+            case STRING:
+                return Types.LONGVARCHAR;
+            case SYMBOL:
+                return Types.NULL;
+            case TIMESTAMP:
+                return Types.NULL;
+            case UNDEFINED:
+                return Types.NULL;
+        }
+        throw new SQLException("Unknown BSON type: " + bsonTypeEnum + ".");
+    }
+
+    @Override
+    public String toString() {
+        return "Column{"
+                + "database='"
+                + database
+                + '\''
+                + ", table='"
+                + table
+                + '\''
+                + ", tableAlias='"
+                + tableAlias
+                + '\''
+                + ", column='"
+                + column
+                + '\''
+                + ", columnAlias='"
+                + columnAlias
+                + '\''
+                + ", bsonType="
+                + bsonType
+                + '}';
+    }
+
+    @Override
+    public boolean isPolymorphic() {
+        return false;
+    }
+
```

Comment:
Since the POJO codec isn't going to call constructors, we need to compute these 3 pieces of data at runtime rather than at construction time as we do in `MongoSQL`. We're probably going to deprecate `MySQL` anyway, so I'm not too worried about it.